README.md

```diff
@@ -1,5 +1,71 @@
-chef-acceptance
----------------
+`chef-acceptance` helps develop and run acceptance tests from your laptop and from Chef Delivery.
 
-`chef-acceptance` is an experimental framework for running acceptance tests
-within a Chef product's source project.
+## Why
+
+`chef-acceptance` makes it easy to develop acceptance tests for a project. You can run any type of suites for your project (test-kitchen, pedant, chef provisioning, rspec, etc...) but it enforces a structure so that you can run all of your different acceptance tests in the same way. It also gives you a CLI interface so that you can build your tests without breaking your pipeline until they are ready.
+
+## Usage
+
+Add `./acceptance` directory to your project with the following directory structure:
+
+```
+acceptance
+├── lamont-ci
+│   ├── .acceptance
```

Comment:
I think we actually want `.acceptance/acceptance-cookbook/*`. This maps to the shape of things in Delivery and it also allows us to drop a `.acceptance/config.json` file sometime in the future. likewise I would probably move the actual `lamont-ci` cookbook to `acceptance/lamont-ci/cookbooks/lamont-ci/*`. Seems a little excessive in this example but as we support more complicated things (e.g. test suites not in cookbooks) it will make more sense.